src/main/tray/tray.ts

```diff
@@ -85,9 +86,10 @@ export function setupTray(iconTheme: string) {
 
     trayIcon.setToolTip(app.name);
     trayIcon.on('click', () => {
-        if (WindowManager.mainWindow!.isVisible()) {
-            WindowManager.mainWindow!.blur(); // To move focus to the next top-level window in Windows
-            WindowManager.mainWindow!.hide();
+        const mainWindow = MainWindow.get(true)!;
+        if (mainWindow.isVisible()) {
+            mainWindow.blur(); // To move focus to the next top-level window in Windows
```

Comment:
is this a windows only thing? are there any side effects in other OS' ? should this be wrapped under a platform only condition?

Reply:
Another one that was just migrated, I don't remember the original context. Gonna leave it for now.